spec/controllers/cafe_works_controller_spec.rb

```diff
@@ -84,23 +84,23 @@
       context 'with no user' do
         it 'update worker' do
           patch :update_worker,
-                {
-                    id: cwork_access.to_param,
-                    cafe_work: attributes_for(:assignee, :test, access_code: cwork_access.access_code)
-                }
-          cwork_access.reload
+            {
+            id: cwork_access.to_param,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.